.php-cs-fixer.dist.php

```diff
@@ -56,9 +49,5 @@
         'ternary_to_null_coalescing'               => true,
         'use_arrow_functions'                      => true,
         'void_return'                              => true,
-    ])->setFinder(
-        PhpCsFixer\Finder::create()
-            ->exclude('vendor')
-            ->in(__DIR__)
-    )
+    ])->setFinder($finder)
```

Comment:
Removing double finder.

Reply:
That i missed that is a bit embarrassing lol, nice catch

Reply:
Do not be embarrassed. I'm training my eye to spot errors in the last 10 years (I'm not writing developing code)